samples/02.react-video/src/pages/MeetingStage.jsx

```diff
@@ -21,51 +24,63 @@ const MeetingStage = () => {
   const videoElement = useRef();
   // MediaSynchronizer ref
   const synchronizer = useRef();
+  // Join error
+  const [error, setError] = useState();
 
   // Initial setup when context is returned
   useEffect(() => {
     (async function () {
-      // Set the initial video src for the player element
-      videoElement.current.src = initialMediaItem.current.src;
+      try {
```

Comment:
should you only try/catch around joinContainer? or do you just want to catch all errors?

Reply:
Was thinking of catching all errors for this sample, since it also includes `await mediaSession.start()`.